samples/client/petstore/kotlin-retrofit2/src/main/kotlin/org/openapitools/client/infrastructure/ApiClient.kt

```diff
@@ -45,6 +45,17 @@ class ApiClient(
                 level = HttpLoggingInterceptor.Level.BODY
             })
     }
+    
+    private var okHttpClient?: OkHttpClient = null
+    
+    constructor(
```

Comment:
@tgerth we don't need a new constructor, you can add the `private val okHttpClient : OkHttpClient? = null` to the constructor at line 18, because it has a default value.
And with this change, you can remove this constructor, and the var in line 49.